plugins/out_es/es.c

```diff
@@ -597,8 +599,14 @@ void cb_es_flush(void *data, size_t bytes,
             ret = elasticsearch_error_check(c);
             if (ret == FLB_TRUE) {
                 /* we got an error */
-                flb_warn("[out_es] Elasticsearch error\n%s",
-                         c->resp.payload);
+                if (ctx->trace_error) {
+                    /*
+                     * Trace the payload that caused the problem. Use fprintf
+                     * instead of flb_error to avoid truncation
+                     */
+                    fprintf(stderr, "[out_es] error: Input\n%s\nOutput\n%s\n",
```

Comment:
how big is the payload that generate truncation ?

Reply:
about 950 bytes or so.

this isn't the important part of the pr tho :)

Reply:
It's important when the current logging interface is blocking debugging. I will increase the buffer size for the logger, please switch the PR code to flb_error().

Reply:
I don't think there is a bounded upper limit. I'll switch to flb_error(), but here this is `trace` for a very specific case, and, the ES output can be KB in size I suppose.